src/test/java/com/wikia/webdriver/testcases/signuptests/SignUpTests.java

```diff
@@ -222,4 +224,21 @@ public void SignUp_008_signupJapaneseUser() {
     preferences.selectTab(tabNames.EMAIL);
     preferences.verifyEmailMeSection();
   }
+
+  @BeforeMethod(alwaysRun = true)
+  public void setEnvironmentCredentials() {
```

Comment:
this code is duplicated. Test body is not the place for environment decision making block